### ISSUE TITLE
mAP not use pre-trained model

### ISSUE THREAD

**Issue by choasup** [smallcorgi/Faster-RCNN_TF#106]
Dose anyone try to train model not use pre-trained model? My mAP is very poor, where is the problem?TF init weights? 

`
AP for aeroplane = 0.032
AP for bicycle = 0.0193
AP for bird = 0.0110
AP for boat = 0.0026
AP for bottle = -1.0000
AP for bus = 0.0164
AP for car = 0.1464
AP for cat = 0.0370
AP for chair = 0.0194
AP for cow = 0.0055
AP for diningtable = 0.0107
AP for dog = 0.0425
AP for horse = 0.1420
AP for motorbike = 0.1233
AP for person = 0.0693
AP for pottedplant = 0.0011
AP for sheep = -1.0000
AP for sofa = 0.0182
AP for train = 0.0188
AP for tvmonitor = 0.0083
Mean AP = -0.0638
`

**Comment by jeandut** [comment]
I get an even worse mAP by simply running with the default parameters every item is at AP=-1 , I made sure no overflow occured during training and I tried with two different learning rate and starting points any idea what could be the issue ? It seems the loss is not converging I seem to remember seeing an issue on unstable training in this repo.
 For the time being I am writing a script to check RoI gradients as the pretrained model working well means that the forward is indeed ok.  
Otherwise I do not know maybe trying to fiddle even more with the learning rate, momentum and inits...

**Comment by jeandut** [comment]
The bad mAP I got was because of a conflict between concurrent saved files I succeeded in achieving 64 mAP by dividing the learning rate by 10 to avoid overflow.

**Comment by ChiefGodMan** [comment]
Can you explain what is conflict between concurrent saved files ? The ckpt files are auto saved, how do we control them ? Please give more details. Thanks!

**Comment by Sh1p-wy** [comment]
same issue.how do you solve this problem? please

**Comment by Zealoe** [comment]
can anyone explain it? I get the same issue.....
